Sources/TuistKit/Commands/Cache/CacheOptions.swift

```diff
@@ -24,9 +24,13 @@ struct CacheOptions: ParsableArguments {
 
     @Option(
         name: .long,
-        help: "Output type of xcframeworks when --xcframeworks is passed (device/simulator)"
+        help: "Output type of xcframeworks when --xcframeworks is passed (device/simulator/all)"
```

Comment:
Add `completion: .list["device", "simulator"]`.
All should not be a valid value, but just be the default if no value is provided

Reply:
Oh thanks, I don't even know about that `completion:`